simulator/src/metadata.json

```
@@ -118,42 +118,42 @@
             {"name":"DigitalLed", "label":"Digital Led"},
             {"name":"VariableLed", "label":"Variable Led"},
             {"name":"HexDisplay", "label":"Hex Display"},
-            {"name":"SevenSegDisplay", "label":"Seven Segment Display"},
-            {"name":"SixteenSegDisplay", "label":"Sixteen Segment Display"},
+            {"name":"SevenSegDisplay", "label":"Seven Segment Display", "type": "IC7447"},
+            {"name":"SixteenSegDisplay", "label":"Sixteen Segment Display", "type": "MAX6954"},
             {"name":"SquareRGBLed", "label":"Square RGB Led"},
             {"name":"RGBLedMatrix", "label":"RGB Led Matrix"}
         ],
         "Gates": [
-            {"name":"AndGate", "label":"And Gate"},
-            {"name":"OrGate", "label":"Or Gate"},
-            {"name":"NotGate", "label":"Not Gate"},
-            {"name":"XorGate", "label":"Xor Gate"},
-            {"name":"NandGate", "label":"Nand Gate"},
-            {"name":"NorGate", "label":"Nor Gate"},
-            {"name":"XnorGate", "label":"Xnor Gate"}
+            {"name":"AndGate", "label":"And Gate", "type": "IC7408"},
+            {"name":"OrGate", "label":"Or Gate", "type": "IC7432"},
+            {"name":"NotGate", "label":"Not Gate", "type": "IC7404"},
+            {"name":"XorGate", "label":"Xor Gate", "type": "IC7486"},
+            {"name":"NandGate", "label":"Nand Gate", "type": "IC7400"},
+            {"name":"NorGate", "label":"Nor Gate", "type": "IC7402"},
+            {"name":"XnorGate", "label":"Xnor Gate", "type": "IC74266"}
         ],
         "Decoders & Plexers": [
-            {"name":"Multiplexer", "label":"Multiplexer"},
-            {"name":"Demultiplexer", "label":"Demultiplexer"},
-            {"name":"BitSelector", "label":"Bit Selector"},
+            {"name":"Multiplexer", "label":"Multiplexer", "type": "IC74157"},
+            {"name":"Demultiplexer", "label":"Demultiplexer", "type": "NL7SZ19"},
+            {"name":"BitSelector", "label":"Bit Selector", "type": "SN74LVC2G157"},
             {"name":"MSB", "label":"MSB(Most Significant Bit)"},
             {"name":"LSB", "label":"LSB(Least Significant Bit)"},
-            {"name":"PriorityEncoder", "label":"Priority Encoder"},
+            {"name":"PriorityEncoder", "label":"Priority Encoder", "type": "IC74147"},
             {"name":"Decoder", "label":"Decoder"}
         ],
         "Sequential Elements": [
-            {"name":"DflipFlop", "label":"D flip Flop"},
-            {"name":"Dlatch", "label":"D latch"},
+            {"name":"DflipFlop", "label":"D flip Flop", "type": "IC74LS74"},
```

```suggestion
            {"name":"DflipFlop", "label":"D flip Flop", "type": "IC7474"},
```
here LS stands for - low power Schottky diode, so the IC number is basically 7474 so please keep it as 7474 only and also change others as well.